src/top/api/main.py

```diff
@@ -58,7 +58,7 @@ async def health():
 async def get_openid_configuration() -> OidcConfiguration:
     """The OpenID discovery endpoint."""
     service_url = str(CONFIG.service_url).rstrip("/")
-    root_path = str(CONFIG.api_root_path).strip("/") + "/"
+    root_path = "/" + str(CONFIG.api_root_path).strip("/") + "/"
```

Comment:
If `api_root_path` is empty or just a slash, you get a duplicated slash.

This should work:

```python
root_url = str(CONFIG.service_url).rstrip("/") + "/"
root_path = str(CONFIG.api_root_path).strip("/")
if root_path:
    root_url += root_path + "/"
```

The `service_url` variable is not needed then.

Reply:
True, this looks better.

Reply:
Yes, looks better indeed.